zetta_utils/mazepa_layer_processing/common/subchunkable_apply_flow.py

```diff
@@ -330,6 +332,12 @@ def build_subchunkable_apply_flow(  # pylint: disable=keyword-arg-before-vararg,
     else:
         allow_cache_up_to_level_ = allow_cache_up_to_level
 
+    if gap is not None and Vec3D(*gap) != Vec3D[int](0, 0, 0):
+        if any(v != 0 for v in processing_blend_pads_[0]):
+            raise ValueError("`blend_pads` at top level must be zero when `gap` is used.")
+        if shrink_processing_chunk:
+            raise ValueError("`shrink_processing_chunk` cannot be used with nonzero `gap`.")
+
```

Comment:
Missing check that `gap` is evenly divisible by 2 in all dimensions.

Also check for `expand_bbox_backend` / `expand_bbox_resolution` and raise as in the case of `shrink_processing_chunk`.